cli/container.go

```diff
@@ -131,18 +135,61 @@ func (c *container) config() (*types.ContainerCreateConfig, error) {
 		}
 	}
 
+	// parse port binding
```

Comment:
In the future, we could encapsulate the following code block into a function like `parsePortMapping`

```
func parsePortMapping(ports []string)(ports [string]nats, portBindings xxxx)
```